_docs/10_general/01_introduction.md

```diff
@@ -4,10 +4,9 @@ category: general
 permalink: /
 ---
 
-In this documentation you will find most of our features explained and find instructions on how to enable certain features.
+Whether you’re a startup or a global enterprise, learn how to integrate with Simple Analytics to move to privacy friendly analytics for your visitors without needed consent.
```

Comment:
Should it be "needed consent" or "needing consent"?

Reply:
Let me know what this should be @jakejarvis so I can merge your PR.

Reply:
Hmm, probably "needing consent" in that case but how does something like this sound split into two sentences?

"Whether you’re a startup or a global enterprise, learn how to integrate with Simple Analytics for privacy friendly analytics. No consent required from your visitors!"